cocotb/utils.py

```diff
@@ -137,6 +138,7 @@ def get_sim_steps(
             ``'step'`` means *time* is already in simulation time steps.
         round_mode: String specifying how to handle time values that sit between time steps
             (one of ``'error'``, ``'round'``, ``'ceil'``, ``'floor'``).
+        tolerance: Maximum relative error allowed when *round_mode* is ``'error'``.
```

Comment:
To match numpy, the name `rtol` might be better.

Reply:
OTOH, the Python standard library calls it `rel_tol` (https://docs.python.org/3/library/math.html#math.isclose).